docs/releases/sdlc-compliance.template.md

```diff
@@ -39,10 +39,6 @@ Overview:
 
 Assumptions and attestations:
 
-1. Internal processes are used to ensure CVEs are identified and mitigated within SLAs.
+- Internal processes are used to ensure CVEs are identified and mitigated within SLAs.
 
-2. The Dependency document does not specify third party OSS CVEs fixed by the release and the date we discovered them.
-
-3. There is no CycloneDX field for original/modified CVSS scor or discovery date. The `x-` prefix indicates this.
-
-3. Assumption: We can include the SBOMs as links to read-only files on S3. The links can be included as metadata or text file links in release artifacts e.g. as labels on OCI containers.
\ No newline at end of file
+- All Operator images are signed by MongoDB, with signatures stored at `docker.io/mongodb/signatures`.
```

Comment:
We probably need to document a way to verify the signatures (a set of commands typically ran by a user) and put it somewhere in the project docs. You can probably copy-paste it from DOCSP-39646.

Reply:
We have that already here, check out line 28, which links to:
[docs/dev/signed-images.md](https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/docs/dev/signed-images.md#image-signature-verification)

Reply:
Missed it. Approving! Thanks!